--- kphone/configure.in.orig	2004-04-01 17:56:26.000000000 +0200
+++ kphone/configure.in	2004-05-30 14:18:55.712259362 +0200
@@ -89,7 +89,7 @@
   AC_DEFINE(QT_THREAD_SUPPORT)
 fi
 
-CFLAGS="-I$ac_cv_qt_inc -Wall"
+CFLAGS="$CFLAGS -I$ac_cv_qt_inc -Wall"
 LIBS="-L$ac_cv_qt_lib -l$ac_cv_qt_libname -L/usr/local/lib -L/usr/X11R6/lib"
 
 dnl Check for X libraries.
@@ -142,7 +142,6 @@
          CFLAGS="$CFLAGS -g -DDEBUG"
 AC_MSG_RESULT(yes)
 else
-CFLAGS="$CFLAGS -O3"
 AC_MSG_RESULT(no)
 fi
 
