diff -Naur kphone/configure.in kphone-s/configure.in
--- kphone/configure.in	Thu Nov  4 10:36:47 2004
+++ kphone-s/configure.in	Thu Nov 11 11:09:33 2004
@@ -89,7 +89,7 @@
   AC_DEFINE(QT_THREAD_SUPPORT)
 fi
 
-CFLAGS="-I$ac_cv_qt_inc -Wall"
+CFLAGS="$CFLAGS -I$ac_cv_qt_inc -Wall"
 LIBS="-L$ac_cv_qt_lib -l$ac_cv_qt_libname -L/usr/local/lib -L/usr/X11R6/lib64 -L/usr/X11R6/lib"
 
 dnl Check for X libraries.
@@ -142,7 +142,6 @@
          CFLAGS="$CFLAGS -g -DDEBUG"
 AC_MSG_RESULT(yes)
 else
-CFLAGS="$CFLAGS -O3"
 AC_MSG_RESULT(no)
 fi
 
